### PR TITLE
Handle Lane mappings

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: lib/folio_record.rb [hunk 1]
```diff
@@ -17,10 +17,61 @@ def instance_id
     record.dig('externalIdsHolder', 'instanceId')
   end
 
+  def sirsi_holdings
+    @sirsi_holdings ||= items.map do |item|
```
Comment (thatbudakguy) [inline lib/folio_record.rb:21]: this is kind of weird. at this point I almost feel like we should make a `FolioHolding` that does nothing other than inherit from `SirsiHolding`, just so we stop ending up with odd things like this that we might forget to change the names of

File: spec/integration/compare_sirsi_and_folio_spec.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 require 'spec_helper'
 
 describe 'comparing against a well-known location full of documents generated by solrmarc' do
```
Comment (thatbudakguy) [inline spec/integration/compare_sirsi_and_folio_spec.rb:3]: this describe text doesn't seem right for the test

**Review by thatbudakguy — APPROVED** [review]

File: spec/integration/compare_sirsi_and_folio_spec.rb (hunk 2 above)
Comment (cbeer) [reply spec/integration/compare_sirsi_and_folio_spec.rb:3]: I agree, but I don't think it's appropriate to change the text in this PR?

Comment (thatbudakguy) [reply spec/integration/compare_sirsi_and_folio_spec.rb:3]: fair!

**Pull request merged**